src/controllers/todo/todo.controller.ts

```diff
@@ -3,81 +3,79 @@ import { Document } from "mongoose";
 import TodoItem from "../../interfaces/todo";
 import { TodoModel } from "./../../models/todo/todo.model";
 
-class TodoController {
-  public create(req: Request, res: Response): void {
-    const todoItem = new TodoModel(req.body);
-    todoItem.save()
-    .then((todo: Document): void => {
-      res.status(201);
-      res.json({ status: true, message: todo});
-    })
-    .catch((error: Error): void => {
-      res.status(500);
-      res.json({ status: false, error });
-    });
-  }
+function create(req: Request, res: Response): void {
+  const todoItem = new TodoModel(req.body);
+  todoItem.save()
+  .then((todo: Document): void => {
+    res.status(201);
+    res.json({ status: true, message: todo});
+  })
+  .catch((error: Error): void => {
+    res.status(500);
+    res.json({ status: false, error });
+  });
+}
 
-  public readAll(req: Request, res: Response): void {
-    TodoModel.find({})
-    .then((todos: Document[]): void => {
-      res.status(200);
-      res.json(todos);
-    })
-    .catch((error: Error): void => {
-      res.status(500);
-      res.json(error.message);
-    });
-  }
+function readAll(req: Request, res: Response): void {
+  TodoModel.find({})
+  .then((todos: Document[]): void => {
+    res.status(200);
+    res.json(todos);
+  })
+  .catch((error: Error): void => {
+    res.status(500);
+    res.json(error.message);
+  });
+}
 
-  public readOne(req: Request, res: Response): void {
-    TodoModel.findOne({ _id: req.params.id })
-    .then((todo: TodoItem | null): void | Error => {
-      if (!todo || !Object.keys(todo).length) {
-        throw new Error("TodoItem not found");
-      }
-      res.status(200);
-      res.json(todo);
-    })
-    .catch((error: Error) => {
-      res.status(500);
-      res.json(error);
-    });
-  }
+function readOne(req: Request, res: Response): void {
+  TodoModel.findOne({ _id: req.params.id })
+  .then((todo: TodoItem | null): void | Error => {
+    if (!todo || !Object.keys(todo).length) {
```

Comment:
Don't think you need `!Object.keys...` because mongoose returns null if the document does not exist. Under what circumstance will an object with no properties be returned